extensions/metrics.py

```
@@ -0,0 +1,189 @@
+from __future__ import annotations
+
+import asyncio
+import logging
+import time
+from typing import Any
+
+import discord
+from discord.ext import commands
+
+from metrics.collector import collect_system_metrics
+from metrics.definitions import (
```

Flake8 (7.3.0)

[error] 'metrics.definitions.message_processing_latency' imported but unused

(F401)